tox.ini

```diff
@@ -1,13 +1,13 @@
 [tox]
-envlist=py3{4,5,6},docs
+envlist=py3{6},docs
 skip_missing_interpreters=True
 
 [testenv]
 deps=-rrequirements/tests.txt
 commands=
-    isort --check-only --diff -p shaarli_client
+    isort --check --diff shaarli_client
     pycodestyle
-    pydocstyle
+    pydocstyle --ignore=D203,D213,D401,D400,D415
```

Comment:
exclusions should be added to the existing https://github.com/shaarli/python-shaarli-client/blob/master/setup.cfg generic configuration file